packages/default/scss/multiselect/_layout.scss

```diff
@@ -4,106 +4,198 @@
     // Base
     .k-multiselect {
         @include border-radius();
-        width: 100%;
+        width: $input-default-width;
```

Comment:
Hello, I believe this has broken the previous design.

It use to be 100%, but now is `$input-default-width` (which for me is 12.4em)

Reply:
Hi there,

Thank you for bringing this to our attention.
It should be fixed in [this PR](https://github.com/telerik/kendo-themes/pull/1109).